contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.1.0--2.2.0.sql

```diff
@@ -161,6 +161,49 @@ CREATE OR REPLACE VIEW information_schema_tsql.COLUMN_DOMAIN_USAGE AS
 
 GRANT SELECT ON information_schema_tsql.COLUMN_DOMAIN_USAGE TO PUBLIC;
 
+/*
+* KEY COLUMN USAGE
+*/
+
+CREATE OR REPLACE VIEW information_schema_tsql.key_column_usage
```

Comment:
what release these are part of? this is upgrade script for lama to manatee. If it is for narwhal, we should create a new upgrade script (I am not sure if branch is open for Narwhal dev or not)

Reply:
This is intended for narwhal. We will have to wait for the open source branches to open for the next release. We will leave these open and document our work until that point.